packages/terra-list/src/ListSectionHeader.jsx

```diff
@@ -73,6 +85,8 @@ const ListSectionHeader = ({
   onSelect,
   refCallback,
   title,
+  role,
+  isDisabled,
   ...customProps
 }) => {
   const sectionHeaderClassNames = cx([
```

Comment:
Sections/Groups within lists can get complicated. With some further research we’ll need

The role of "heading" needs to move to a div or h# tag internal to the li.

Next, I believe there are 4 scenarios we need to validate for, so tests for each might be appropriate. I’ve listed some considerations for each.

Scenario 1: Default List with Static Section header
  - li with role of "presentation" or "none
  - A child h# tag containing the title

Scenario 2: Default List with Collapsible Section Header
  - li with role of "presentation"
  - A child interactive element, perhaps a button, containing a child h# tag containing the title
  - A child ul containing the relation options? Or simple the remaining grouped li’s with role option contained within the presentation?

Scenario 3: Interactive List with Static Section Header
  - li with role of "presentation"
  - A child h# tag containing the title

Scenario 4: Interactive List with Collapsible
  - li with role of "presentation" or a role of “option”? Needs further testing.
  - A child interactive element, perhaps a button, containing a child h# tag containing the title
  - A child ul containing the relation options? Or simple the remaining grouped li’s with role option contained within the presentation?

With the pending deprecation of the terra-infinite we no longer need to worry about flat list of list items. We may need to evaluate the inclusion of nested ul’s again, I know it had some issues within iOS at the time. Here is a good reference:
https://w3c.github.io/aria-practices/examples/listbox/listbox-grouped.html

Reply:
Moved `role="heading"` to title element. cf4a187
Role= "presentation" is giving accessibility issue with parent having role="listbox", which is required if any or all other items have role as option.

Reply:
Do you have an example of the error? There are other changes, like moving the tab-index, aria-expanded, etc all moving to internal elements. Also potentially moving all child lis inside another ul presentations as well.

Reply:
My guess is the error of having no options at all, when all options are collapse or only the section header is used in the example. There may need to be more testing to figure out the correct roles, but the goal for this is that the section header shouldn't be read as/or totalled in the listbox counts.

Reply:
In the list-section example when we have selectable items, having section role as `option` means that the Voice Over does not read it as heading, but if we have heading as role then there are accessibility violations saying that the role should be option as the list's role is listbox.

Reply:
We definitely don't want the role of 'heading' on the item, and more than likely 'option' isn't the correct role either. You will need to spike on what the proper fix is. Like I mentioned above, combing a presentation with a interior button that has the header inside may be correct for the section header itself, but in the case of all sections collapsed we will no longer have an option present within the 'listbox', so we will either need to revise that strategy to merely hide the list items, or find a way to get the screen reader to still read how many items are within the listbox even with them no longer on the DOM.